base_phone_popup/models/phone_common.py

```diff
@@ -44,26 +43,21 @@ def incall_notify_by_login(self, number, login_list):
         res = self.get_record_from_phone_number(number)
         users = self.env['res.users'].search(
             [('login', 'in', login_list)])
-        logger.debug(
-            'Notify incoming call from number %s to users %s'
+        logger.info(
+            'Notify incoming call from number %s to users IDs %s'
             % (number, users.ids))
         action = self._prepare_incall_pop_action(res, number)
         if action:
-            for user in users:
-                if user.context_incall_popup:
-                    self.sudo(user.id).env['action.request'].notify(action)
-                    logger.debug(
-                        'This action has been sent to user ID %d: %s'
-                        % (user.id, action))
+            title = _('Incoming call')
+            message = _('Call from %s') % number
+            action_link_name = res and res[2] or action['name']
+            users.notify_info(
+                message, title, True, False, action, action_link_name)
```

Comment:
Would be more readable to use the keyword arguments (`title=title, sticky=True, action=action, action_link_name=action_link_name`)

So the reader don't have to read the implementation to know what True and False refers to, also then you don't have to pass `show_reload` (default value)